### PR TITLE
Add is/setVehicleWindowOpen

### PR DESCRIPTION
**Pull request by lopezloo**
So, this basically adds clientside function to control vehicle windows. Also I seen when ped start driveby as driver the window don't open, this fixes that. I redefined windows ID to be 0 - 6 but I'm not sure I do it proper place (game_sa).

### CODE REVIEW

File: MTA10/mods/shared_logic/CClientPed.cpp [hunk 1]
```diff
@@ -5576,12 +5576,37 @@ void CClientPed::SetDoingGangDriveby ( bool bDriveby )
         }
         else if ( bDriveby )
         {
-            bool bRight = ( GetOccupiedVehicleSeat ( ) % 2 == 0 ) ? false : true;
+            char cSeat = GetOccupiedVehicleSeat ( );
+            bool bRight = ( cSeat % 2 == 0 ) ? false : true;
```
Comment (sbx320) [inline MTA10/mods/shared_logic/CClientPed.cpp]: cSeat % 2 == 0 is already a boolean expression.

cSeat % 2 != 0 has the same effect without the ternary operator

**Comment by lopezloo** [comment]
Hm... I can do it but in that case function will return always true, even vehicle don't have specified window.

//edit
Maybe make this always return true? I seen setVehicleDoorState do that.

**Comment by ccw808** [comment]
I think that is acceptable.

**Comment by lopezloo** [comment]
Ok, done.

**Comment by qaisjp** [comment]
Nice work! :+1:

**Pull request merged**